seed/ts-sdk/any-auth/generate-endpoint-metadata/tests/wire/mockAuth.ts

```diff
@@ -0,0 +1,11 @@
+// This file was auto-generated by Fern from our API Definition.
+
+import type { MockServer } from "../mock-server/MockServer";
+
+export function mockAuth(server: MockServer) {
```

Comment:
Instead of generating a `mockAuth` function, generate a `mock{auth_scheme_key_in_pascal_case}`, one for each scheme defined in the IR that requires mocking.

Then in the generated wire tests, call all the necessary mock auth's based on the logic in generators/typescript/sdk/client-class-generator/src/GeneratedSdkClientClassImpl.ts